hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueFactory.kt

```diff
@@ -77,7 +79,7 @@ class HmppsQueueFactory(
         override fun handle(message: org.springframework.messaging.Message<Any>, t: Throwable) {
           // SDI-477 remove this logging when we are comfortable that all is working as expected - instant retries
           log.info("Setting visibility of messageId ${message.headers["id"]} to $errorVisibilityTimeout (to initiate faster retry) after receiving exception ${t.cause?.cause?.cause}")
-          val sqsVisibility = message.headers["Sqs_Visibility"] as QueueMessageVisibility
+          val sqsVisibility = message.headers["Sqs_VisibilityTimeout"] as QueueMessageVisibility
```

Comment:
what's going on here?

Reply:
It appears the name of the property has changed.